stripecardscan-example/src/main/java/com/stripe/android/stripecardscan/example/MainActivity.kt

```diff
@@ -13,12 +21,50 @@ class MainActivity : AppCompatActivity() {
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        val requestQueue = Volley.newRequestQueue(this)
+
         super.onCreate(savedInstanceState)
         setContentView(viewBinding.root)
         setSupportActionBar(findViewById(R.id.toolbar))
 
+        val civLauncher = registerForActivityResult(
+            ActivityResultContracts.StartActivityForResult()
+        ) { result ->
+            viewBinding.scanResultText.text = result.data?.getStringExtra(PARAM_VERIFICATION_RESULT)
+        }
+
+        viewBinding.getPublishableKeyButton.setOnClickListener {
+            requestQueue.add(
+                JsonObjectRequest(
```

Comment:
We started using [Fuel](https://github.com/kittinunf/fuel) for the payment sheet playground and examples. I'm wondering if we should unify that with the other parts of the SDK?